Finder.php

```diff
@@ -134,7 +134,19 @@ public function in($path)
         }
 
         foreach ($path as $p) {
-            $this->_paths[] = $p;
+            if (1 === preg_match('/[\*\?\[\]]/', $p)) {
```

Comment:
Hum, yes and no... `preg_match` return `1` if the pattern match but can return `0` or `false` if it doesn't match or encountered a problem... It's because of that double case that I've used `1` in my test.
What do you think about ?